tools/cli/commands/changelog.js

```diff
@@ -369,8 +369,8 @@ async function addIndirectPlugins( argv, directProjects, changelogFiles ) {
 			for ( const file of changelogFiles.get( proj ) ) {
```

Comment:
Should we update the comment just above?
https://github.com/Automattic/jetpack/blob/080b34119ca27f2b33ccd177bcec27dcbf215d71/tools/cli/commands/changelog.js#L366

Reply:
beb63c04a8